### ISSUE TITLE
New points system should had migrated from karma system!

### ISSUE THREAD

**Issue by antdude** [ruby-rbot/rbot#48]
33792 Apr 24 14:11 karmaplugin.tdb
  5376 May 24 20:42 pointsplugin.tdb

I noticed my two rbots still have karma points. Why didn't the upgraded rbot's point systems use the old karmas to convert/migrate to points? Some of us are sad to lose our old karmas. Can we get them to points please? ;)

**Comment by mattzque** [comment]
yes good point! for some reason I thought that it did that already but that were a bunch of other plugins like factoids.
